client/components/App.jsx

```diff
@@ -14,6 +15,7 @@ class App extends React.Component {
         <>
           <Route path='/' component={TopMenu} />
           <Container style={{ marginTop: 75 }}>
+            <Route path='/' component={CategoriesListDropdown} />
```

Comment:
please put inside switch statements

Reply:
@emilyparkes does it need to be inside the switch statement?